app/src/components/account/UserMenu.vue

```diff
@@ -182,6 +185,7 @@ export default {
         let allAddressesWithAddressRole = await this.getAllAddressesRoles(
           allAddresses.filter(({ address }) => address)
         )
+        this.$store.commit(`setUserAccounts`, allAddressesWithAddressRole)
```

Comment:
🙈

Reply:
:shrug:  how to do it otherwise?

Reply:
You want to recalculate all them in ManageAccounts all over?

Reply:
it's a fine hack as we will move into lunie3. in general you should move data logic into the store if it is used in several places. so we would likely call `getAllAccounts` which would in the store get all the accounts and deliver them to the component.